xapian-core/queryparser/queryparser.lt

```diff
@@ -569,16 +569,14 @@ static void yyStackOverflow(yyParser *yypParser){
 */
 #ifdef XAPIAN_DEBUG_LOG
 static void yyTraceShift(yyParser *yypParser, int yyNewState, const char *zTag){
-  if( yyTraceFILE ){
-    if( yyNewState<YYNSTATE ){
-      LOGLINE(QUERYPARSER, zTag << " '" <<
-			   yyTokenName[yypParser->yystack.back().major] <<
-			   "', go to state " << yyNewState);
-    }else{
-      LOGLINE(QUERYPARSER, zTag << " '" <<
-			   yyTokenName[yypParser->yystack.back().major] <<
-			   "', pending reduce " << yyNewState - YY_MIN_REDUCE);
-    }
+  if( yyNewState<YYNSTATE ){
+    LOGLINE(QUERYPARSER, zTag << " '" <<
+		   yyTokenName[yypParser->yystack.back().major] <<
+		   "', go to state " << yyNewState);
+  }else{
+    LOGLINE(QUERYPARSER, zTag << " '" <<
+		   yyTokenName[yypParser->yystack.back().major] <<
+		   "', pending reduce " << yyNewState - YY_MIN_REDUCE);
```

Comment:
As I wrote in #185:

> The first non-whitespace character of the continuation lines should line up under the `z` of `zTag` as it did before (in an editor at least; in github's diff viewer it appears to be off by one column because the first line a 4 space indent, but the continuation lines start with a tab, and so the leading `-` makes a difference).

You said (in a comment in #185) that you've addressed this, but it is still wrong here.

Reply:
Fixed this issue in a new commit. Aligned the lines below `ztag`.